### PR TITLE
Fix build on Apple Silicon

### PR DESCRIPTION
**Pull request by tylinux**
## Motivation

`brew install facebook/fb/idb-companion` on Apple M1 devices is broken.

## Test Plan

1. Open `idb_companion.xcworkspace` on Apple M1 devices, then `CMD+R` to build `idb_companion`.
2. Or execute `./idb_build.sh idb_companion build` on Apple M1 devices to build `idb_companion`

## Related PRs

No.

### CODE REVIEW

File: idb/cli/main.py [hunk 1]
```diff
@@ -237,7 +239,8 @@ async def gen_main(cmd_input: Optional[List[str]] = None) -> int:
         CommandGroup(
             name="crash",
             description="Operations on crashes",
-            commands=[CrashListCommand(), CrashShowCommand(), CrashDeleteCommand()],
+            commands=[CrashListCommand(), CrashShowCommand(),
+                      CrashDeleteCommand()],
```
[the remark this answers is not in the document]
Comment (tylinux) [reply idb/cli/main.py]: My vscode formatter done this change, I just reverted it.

**Comment by c-ryan747** [comment]
Thanks for adding M1 support @tylinux !

I exited your commit slightly before landing to make sure that the idb cli still functions well on linux

**Comment by jeffreyabarrios** [comment]
Is this fixed already for M1 then? I tried again today and still got this error installing idb-companion:

`** BUILD FAILED **


The following build commands failed:
	CompileC /tmp/idb-companion-20210722-21958-rqven6/idb-1.1.5/build/Build/Intermediates.noindex/idb_companion.build/Debug/idbGRPC.build/Objects-normal/arm64/idb.pb.o /tmp/idb-companion-20210722-21958-rqven6/idb-1.1.5/build/Build/Intermediates.noindex/idb_companion.build/Debug/idbGRPC.build/DerivedSources/idb.pb.cc normal arm64 c++ com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)`

**Comment by dawidchyrzynski** [comment]
Same here. Build is failing on M1.
```
The following build commands failed:
	RuleScriptExecution /tmp/idb-companion-20210801-52632-1dlvlme/idb-1.1.5/build/Build/Intermediates.noindex/idb_companion.build/Debug/idbGRPC.build/DerivedSources/idb.pb.h /tmp/idb-companion-20210801-52632-1dlvlme/idb-1.1.5/build/Build/Intermediates.noindex/idb_companion.build/Debug/idbGRPC.build/DerivedSources/idb.pb.cc /tmp/idb-companion-20210801-52632-1dlvlme/idb-1.1.5/build/Build/Intermediates.noindex/idb_companion.build/Debug/idbGRPC.build/DerivedSources/idb.grpc.pb.h /tmp/idb-companion-20210801-52632-1dlvlme/idb-1.1.5/build/Build/Intermediates.noindex/idb_companion.build/Debug/idbGRPC.build/DerivedSources/idb.grpc.pb.cc /tmp/idb-companion-20210801-52632-1dlvlme/idb-1.1.5/proto/idb.proto normal arm64
(1 failure)

```

**Pull request closed without merging**